code/game/objects/items/devices/busterarm/gasharpoon.dm

```diff
@@ -1,13 +1,13 @@
 /obj/item/clothing/gloves/gasharpoon
-	name = "gas-harpoon"
+	name = "GasHarpoon"
```

Comment:
we don't capitalize these kinds of items in our code

Reply:
ok